editor/src/components/canvas/controls/select-mode/select-mode-hooks.tsx

```diff
@@ -184,41 +184,17 @@ function collectSelectableSiblings(
   childrenSelectable: boolean,
   lockedElements: LockedElements,
 ): Array<ElementPath> {
-  let siblings: Array<ElementPath> = []
+  let siblingsChildrenAndAncestors: Array<ElementPath> = []
   Utils.fastForEach(selectedViews, (view) => {
     function addChildrenAndUnfurledFocusedComponents(paths: Array<ElementPath>) {
```

Comment:
now that the diff is _already_ unreadable, we may as well improve the code quality here! I think we could kill this function, and we could also kill the `let siblingsChildrenAndAncestors` and make this part functional

Reply:
Yeah good point, I have tidied this all up now so the code should be a lot clearer 👍